tests/Test/Proto/Generate/Name/Gen.hs

```diff
@@ -13,11 +13,12 @@ module Test.Proto.Generate.Name.Gen
   )
 where
 
+import Control.Applicative (liftA2)
```

Comment:
Comment only: I guess this is the sort of import that would make warnings-as-errors a pain for hackage maintainers, because it is sometimes redundant and sometimes required, depending upon the version of `base`.